src/foam/nanos/logger/DiscardLogger.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.logger',
+  name: 'DiscardLogger',
+  extends: 'foam.nanos.logger.ProxyLogger',
+ 
+  properties: [
+    {
+      name: 'logError',
+      class: 'Boolean',
+      value: true
```

Comment:
Do you need to set the javaFactory: also?

Reply:
No, don't. Have removed it.